<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Info</class>
 <widget class="QWidget" name="Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="save_information">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save_Information</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>266</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X_coordinate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="X_coordinate"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>212</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y_coordinate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Y_coordinate"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineWidth"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineHeight"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
